docs/reference/changelog.rst

```diff
@@ -2,7 +2,17 @@
 Changelog
 *********
 
+0.8.0 (2025-04-16)
```

Comment:
@tigarmo should we include the Rockcraft change in 0.8.0?

Reply:
yes please, if you can